yat-docs/docs/partner-guides/1_crypto_wallets.md

```diff
@@ -0,0 +1,304 @@
+
+&nbsp;  
+As the first step of the integration for an existing partner app user, the partner app makes a call to the /process endpoint to:
+
+1. Create and activate a new Yat user
+2. Allow the user to purchase a Yat
+3. Assign the Yat to the newly created user
+
+**Service URL:** https://y.at/process
+
+**Method:** `POST`
+
+**Headers:**
+
+- `X-Bypass-Token:` Contains an authentication token for the partner. To be delivered to the partner by the Yat team.
+
+**Body:** A JSON object containing the id and password of the user. This pair can later be used to authenticate the user with the Yat API. `alternate_id` is a deterministic public value based on the unique wallet. `password` is a deterministic secret that can be any secret that is signed by the secret key for the wallet.
+
+``` json
+{
+    "alternate_id": "user_id",
+    "password": "user_password"
+}
+```
+
+**Response:** A JSON object that contains the user authentication info (`access_token` and `refresh_token`), user details (Yat user `id`, `alternate_id` that was sent by the partner, and more).
+
+``` json
+{
+    "auth": {
+        "access_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJiNmI2ZGE4MS0yNTBlLTQ1NmUtYTYxYS01ZjQyNmFlOTJhMjIiLCJpc3MiOiJlbW9qaS1pZC10b2tlbi1pc3N1ZXIiLCJleHAiOjE2MjAzNzczMjgsInNjb3BlcyI6WyJjYXJ0OnNob3ciLCJjYXJ0OnVwZGF0ZSIsInVzZXI6Y3JlYXRlQXBpS2V5IiwiZWRpdGlvbjpyZWFkIiwiZW1vamk6OnRyYW5zZmVyIiwibG9vdGJveDp1c2UiLCJvcmRlcjpyZWFkU2VsZiIsIm9yZ2FuaXphdGlvbkxpc3Q6cmVhZCIsInBheW1lbnRNZXRob2Q6ZGVzdHJveSIsInBheW1lbnRNZXRob2Q6cmVhZCIsInBheW1lbnRNZXRob2Q6c2V0RGVmYXVsdCIsInVzZXI6ZGVsZXRlU2VsZiIsInVzZXJJbnRlcmVzdDpkZWxldGUiLCJ1c2VySW50ZXJlc3Q6cmVhZCIsInVzZXJJbnRlcmVzdDp3cml0ZSIsInVzZXI6d3JpdGVTZWxmIl0sImlzc3VlZCI6MTYyMDM3NjQyOCwiYWN0aXZlMmZhIjowfQ.FZBzoe8Zky9Pl7WCNXwg5KphVO4FNTKxJKX87w9sHW0",
+        "refresh_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJiNmI2ZGE4MS0yNTBlLTQ1NmUtYTYxYS01ZjQyNmFlOTJhMjIiLCJpc3MiOiJlbW9qaS1pZC10b2tlbi1pc3N1ZXIiLCJleHAiOjE2MjMwNTQ4MjgsInNjb3BlcyI6WyJ0b2tlbjpyZWZyZXNoIl0sImlzc3VlZCI6MTYyMDM3NjQyOCwiYWN0aXZlMmZhIjowfQ.8Tr8zLbMLNzleG9KL0zJihBI6cd5V8rXPIOjQjcB_WI",
+        "requires_2fa": null
+    },
+    "user": {
+        "id": "b6b6da81-250e-456e-a61a-5f426ae92a22",
+        "alternate_id": "keith24",
+        "is_active": true,
+        "pubkeys": [
+            "ac7f4164608a0b75a3d97da9bd27dc957c2ee80462875daf5e7ba18bf3dce545"
+        ],
+        "free_limit": 0,
+        "remaining_free_emoji": 0
+    },
+}
+```
+
+### Redirection of the User to the Yat Web Application
+
+The partner application user needs to get redirected to the Yat website to Get a Yat (Flow [1A](#1a-get-a-yat-custodial-wallets) & [1B](#1b-get-a-yat-non-custodial-wallets)) or Connect a Yat (Flow [2](#2-connect-existing-yats)). A number of query parameters are necessary for the Yat web application to correctly identify the user and link any necessary data to the user's Yat.
+
+Web App URL: `https://y.at/partner/{partner_path}`
```

Comment:
hi there! sorry not sure how much this might change the structure of how this is written now, but just wanted to note that this URL is correct for flows 1a and 1b, but for flows 2a and 2b the URL is `https://y.at/partner/{partner_path}/link-email`